study.md

```diff
@@ -111,15 +121,17 @@ The array should contain two strings, "drinking beer" and "propane and propane
 accessories".
 
 ```javascript
-// your answer here
+hank.push({middle_name:'Rutherford'});
```

Comment:
`hank` is an object and `push` isn't an `Object` method.  Even if it were possible, you would be adding an object with a key named `middle_name` and a value of `'Rutherford'`.  This should be `hank['middle name'] = 'Rutherford';`.